src/templates/base/2019/base_ebook.html

```diff
@@ -27,125 +32,101 @@
-                    </a>
-                    {% else %}
-                    <a href="#{{ chapter_config.chapter }}">
-                      {{ self.chapter() }} {{ chapter_config.chapter }}: {{ localizedChapterTitles[chapter_config.title] if localizedChapterTitles[chapter_config.title]|length else chapter_config.title }} <span class="not-translated">({{ self.translation_not_available() }})</span>
-                    </a>
-                    {% endif %}
-                  {% endif %}
-                  </div>
-              {% endfor %}
-            </div>
-          </section>
-        {% endfor %}
-        <section class="part">
-          <h2 class="part-name">{{ self.appendix() }}</h2>
-          <div class="chapters">
-            <div class="chapter">
-              <a href="#methodology">{{ self.methodology_title() }}</a>
-            </div>
-            <div class="chapter">
-              <a href="#contributors">{{ self.contributors_title() }}</a>
-            </div>
+          <div class="chapter">
+            <a href="#contributors">{{ self.contributors_title() }}</a>
           </div>
-        </section>
-      </section>
-
-      <section id="foreword">
-        <h1>{{ self.foreword_title() }}</h1>
-        {{ self.foreword() | replace('/%s/%s/contributors#' % (lang, year), '#contributors-') }}
-      </section>
-
-    {{ self.chapters() }}
-
-    <section class="chapter" id="methodology">
-      <div class="subtitle">Appendix A</div>
-      {{ get_ebook_methodology(lang,year) | safe }}
-    </section>
-
-    <section class="chapter" id="contributors">
-      <div class="subtitle">Appendix B</div>
-    
-        <h1 class="title title-lg">
-          {{ self.contributors_title() }}
-        </h1>
-        <div class="team">
-            <img src="/static/images/avatars/1.jpg" alt="" height="100" width="100" loading="lazy" />
-            <img src="/static/images/avatars/8.jpg" alt="" height="100" width="100" loading="lazy" />
-            <img src="/static/images/avatars/4.jpg" alt="" height="100" width="100" loading="lazy" />
-            <img src="/static/images/avatars/0.jpg" alt="" height="100" width="100" loading="lazy" />
-            <img src="/static/images/avatars/11.jpg" alt="" height="100" width="100" loading="lazy" />
-            <img src="/static/images/avatars/9.jpg" alt="" height="100" width="100" loading="lazy" />
         </div>
-        <p>{{ self.contributors_description() }}</p>
-        <ul class="contributors">
-        {% for id, contributor in config.contributors.items() %}
-        <li class="contributor" id="contributors-{{ id }}">
-          <img class="contributor-avatar" src="{{contributor.avatar_url}}" height="100" width="100" alt="{{ contributor.name }} avatar" />
-          <div>
+      </div>
+    </div>
+  </section>
+
+  <section id="foreword">
```

Comment:
Looks like every page after the foreword has "Foreword" as the header text.
![image](https://user-images.githubusercontent.com/1120896/82126514-47d2f280-977b-11ea-9bb2-314260465436.png)